client/src/components/continuous/continuous.js

```diff
@@ -68,8 +68,15 @@ class Continuous extends React.Component {
               const summary = obsAnnotations.col(key).summarize();
               const isColorField =
                 key.includes("color") || key.includes("Color");
+              const nonFiniteExtent =
+                summary.min === undefined || summary.max === undefined;
```

Comment:
I believe this line actually suppresses display of any continuous annotation with a nonfinite `min` _OR_ a nonfinite `max`?

Semantic difference from the PR description (suppress if min _and_ max are nonfinite -- i.e., _all_ values are `NaN`), but how does JS handle these values? 
If it mirrors python behavior, then this may throw an error if there are _any_ nonfinite values?

Reply:
typo in the PR description; code is correct.  Great catch - I will fix the PR!

Essential idea is that you can't build a histogram unless you know the extent (ie, both min and max)